mantidproject/mantid
# Crystal field intensity in milibarn per steradian

This PR changes the Crystal Field code to output the (energy) integrated intensity in mb/sr rather than in barn (the FOCUS default) because ISIS normalised counts is usually given in mb/sr/meV. 

**To test:**

Run any of the examples in the documentation and verify that the new calculated intensity is `1000/4/pi` greater than previously. E.g. From http://docs.mantidproject.org/nightly/interfaces/CrystalFieldPythonInterface.html

```
cf = CrystalField('Ce', 'C2v', B20=0.37737, B22=3.9770, Temperature=44)
print cf.getPeakList()
```

previously gave:

```
[[  0.00000000e+00   2.44006198e+01   4.24977124e+01   1.80970926e+01  -2.44006198e+01]
 [  2.72327784e+00   1.10973434e+00   6.33885503e-02   2.15078093e-03   1.77951652e-03]]
```

but should now give:

```
[[  0.00000000e+00   2.44006198e+01   4.24977124e+01   1.80970926e+01 -2.44006198e+01]
 [  2.16711565e+02   8.83098530e+01   5.04430056e+00   1.71153708e-01  1.41609425e-01]]
```

Fixes #17652 .

Documentation has been updated to give the units of the calculated intensity.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
